force-app/main/default/permissionsets/Mainly_Digital_Candidate.permissionset-meta.xml

```diff
@@ -4,7 +4,7 @@
         <application>Mainly_Digital</application>
         <visible>true</visible>
     </applicationVisibilities>
-    <description>Designed for new joiners to the company. This permission set enables candidates to access and progress through their specific onboarding journey.</description>
```

Comment:
Worth mentioning MainlyDigital in your desc, you've done so for the Admin description so just for some consistency. Even more important in this case, as this permission set is only for the application whereas admins can be both MD and the wider platform. You should do the same for manager, as it's specific to the platform.